app/Http/Controllers/Api/UsersController.php

```diff
@@ -736,6 +737,23 @@ public function getCurrentUserInfo(Request $request) : array
         return (new UsersTransformer)->transformUser($request->user());
     }
 
+    /**
+     * Display the EULAs accepted by the user.
+     *
+     *  @param \App\Models\User $user
+     *  @param \App\Http\Transformers\ActionlogsTransformer $transformer
+     *  @return \Illuminate\Http\JsonResponse
+     *@since [v8.1.16]
+     * @author [Godfrey Martinez] [<gmartinez@grokability.com>]
+     */
+    public function eulas(User $user, ActionlogsTransformer $transformer)
+    {
+        $eulas = $user->eulas;
+        return response()->json(
+            $transformer->transformActionlogs($eulas, $eulas->count())
```

Comment:
Would this API endpoint potentially leak additional information the user shouldn't see?

Reply:
opening a thread in slack